instance_role.tf

```diff
@@ -0,0 +1,44 @@
+# IAM assume role policy document for the S3 certboto certificate
+# access role to be used by the IPA master EC2 instance
+data "aws_iam_policy_document" "assume_role_doc" {
+  statement {
+    effect = "Allow"
+
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+  }
+}
+
+# The S3 certificate access role to be used by the IPA master EC2
+# instance
+resource "aws_iam_role" "ipa" {
+  assume_role_policy = data.aws_iam_policy_document.assume_role_doc.json
```

Comment:
I'm not sure how much it matters, but it might be nice to name this role and the instance profile below with something meaningful so that it can be easily identified in the AWS console.  I do something similar in the [pca-terraform project](https://github.com/cisagov/pca-terraform/blob/develop/guacamole_iam.tf#L11).

Reply:
Suggestion implemented in commit 5bb8ca3.